File: android/src/main/kotlin/com/sabiantech/sabian_native_common/structures/Loader.kt
package com.sabiantech.sabian_native_common.structures

data class Loader(var title: String = "", var message: String = "", var isHidden: Boolean = false) : IMappable {
    override val toMap: Map<String, Any>
        get() = mapOf("title" to title, "message" to message, "isHidden" to isHidden)
}

File: ios/Classes/structures/Loader.swift
//
//  Loader.swift
//  Runner
//
//  Created by bryosabian on 22/11/2023.
//

import Foundation


struct Loader : IMappable {
    
    var title: String = ""
    var message: String = ""
    var isHidden: Bool = false
    
   var toMap: Dictionary<String, Any>{
        get {
            var dictionary = Dictionary<String,Any>()
            dictionary["title"] = title
            dictionary["message"] = message
            dictionary["isHidden"] = isHidden
            return dictionary
        }
    }
}
